heareval/embeddings/task_embeddings.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, module_name: str, model_path: str = None):
             self.model.to(self.device)
         elif isinstance(self.model, tf.Module):
             self.type = TENSORFLOW
-            raise NotImplementedError("TensorFlow embeddings not supported yet.")
+            self.device = 'cpu'
```

Comment:
My understanding is that tensorflow will automatically use a GPU if one is present, unless you tell it specifically to use CPU or some other device. So I don't think we should set this attribute to CPU since that will be incorrect some of the time (i.e. if a GPU is present). If there is a method to get which device will be used, let's use that. Otherwise I would be more comfortable leaving self.device uninitialized.

Unless I'm missing something and there is a deeper reason why this is being set to CPU. In which case that should be documented.

Reply:
Ah yes, "cpu" is a bit misleading here, since it is not used at all by Tensorflow code. I can set it to None, and add a comment about it not being used?